packages/std/src/entry_points.rs

```diff
@@ -0,0 +1,137 @@
+/// This macro generates the boilerplate required to call into the
+/// contract-specific logic from the entry-points to the WASM module.
```

Comment:
Since you say you're a language nerd, the official appreciation of WebAssembly is Wasm (https://webassembly.org/)

Reply:
Oh, haha, thanks :)
I'll keep that in mind

Reply:
addressed in 0f33c07